### PR TITLE
281 document history view

### LINKED ISSUE #281: Simple document history view (mainly for dev)

### PR DESCRIPTION
**Pull request by clemens-msupply**
Adds a simple document history viewer

Closes #281

### CODE REVIEW

**Comment by clemens-msupply** [comment]
Will create a new PR due to conflicts

File: client/packages/system/src/Patient/DocumentHistory/DocumentHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                    textAlign: 'start',
+                    whiteSpace: 'nowrap',
+                  }}
+                >
+                  {h.timestamp}
+                </Typography>
+              </Box>
+            </AccordionSummary>
+            <AccordionDetails>
+              <ReactDiffViewer
```
Comment (andreievg) [inline client/packages/system/src/Patient/DocumentHistory/DocumentHistory.tsx:89]: does this show diff of JSON ? Is this temporary (just for the demos) ?

File: client/packages/system/src/Patient/DocumentHistory/DocumentHistory.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      return undefined;
+    }
+    return history.find(node => node.id === parentId);
+  };
+  const toString = (node: DocumentNode | undefined): string => {
+    if (node === undefined) {
+      return '';
+    }
+    const clone = JSON.parse(JSON.stringify(node));
+    delete clone['__typename'];
```
Comment (andreievg) [inline client/packages/system/src/Patient/DocumentHistory/DocumentHistory.tsx:41]: does this create empty keys "" ?

File: client/packages/system/src/Patient/DocumentHistory/DocumentHistory.tsx (hunk 1 above)
Comment (andreievg) [reply client/packages/system/src/Patient/DocumentHistory/DocumentHistory.tsx:89]: oops, i saw the issue name, never mind

File: client/packages/system/src/Patient/DocumentHistory/DocumentHistory.tsx (hunk 2 above)
Comment (clemens-msupply) [reply client/packages/system/src/Patient/DocumentHistory/DocumentHistory.tsx:41]: It removes the property completely so that it isn't shown at allow

**Pull request closed without merging**